static/app/views/issueList/overview.tsx

```diff
@@ -950,6 +950,9 @@ class IssueListOverview extends Component<Props, State> {
       organization: this.props.organization,
       search_type: 'issues',
       id: savedSearch.id ? parseInt(savedSearch.id, 10) : -1,
+      is_global: savedSearch.isGlobal,
+      query: savedSearch.query,
+      visibility: 'organization',
```

Comment:
you can update to use real visibility. it's returned from the API now

Reply:
ah yeah it is - added